### PR TITLE
Portscan 212

### LINKED ISSUE #212: portscan not working anymore
the previous `php`-call possible with `whatsmyip.org` does not work anymore. I propose we replace it with a simple service on `dedis.ch` that listens on a port. If a host sends a number to that port, the service will try to connect back to the host at the sent port-#. This avoids abuse of the system.

### PR DESCRIPTION
**Pull request by ineiti**
Closes #212

### CODE REVIEW

**Comment by nikkolasg** [comment]
Should we not merge that first into master, then v1.0 ? Since that is beneficial for all versions.

**Comment by ineiti** [comment]
yes, but I'm creating docker-images:

https://hub.docker.com/r/dedis/conode/

and this depends on v1 - so it was simpler to make this on v1 - but I'll do the backport. Or forwardport?

File: app/server.go [hunk 1]
```diff
@@ -318,41 +324,33 @@ func tryConnect(ip, binding network.Address) error {
...
+	buff, err := ioutil.ReadAll(resp.Body)
+	resp.Body.Close()
 	if err != nil {
 		return err
 	}
 
-	if !bytes.Contains(buffer, []byte("1")) {
-		return errors.New("Address unreachable")
+	res := string(buff)
+	if strings.Compare("Open", res) != 0 {
```
Comment (nikkolasg) [inline app/server.go]: It's nitpicking but it seems overly complicated for a string comparison... why not simply `res == "Open"` ?

Comment (nikkolasg) [inline app/server.go]: Even golang says it's clearer and faster to use the operator == in [the doc](https://golang.org/pkg/strings/#Compare).

Comment (ineiti) [inline app/server.go]: oops - I had java on my mind...

**Pull request merged**